### PR TITLE
Add authentication only check for @Secured

### PR DESCRIPTION
**Pull request by CleverUnderDog**
This change allows to add @Secured to any Service/Method without a value specified. Then it only checks if the request is authenticated and sets the authentication accordingly.

### CODE REVIEW

**Comment by jvmlet** [comment]
`authenticated()` method does exactly this while preserving semantics.

**Comment by CleverUnderDog** [comment]
You are right, but you have to do the definition of secured methods by hand via Grp security. It is not possible to archive this with the secured annotation yet.

File: grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/security/GrpcServiceAuthorizationConfigurer.java [hunk 1]
```diff
@@ -74,8 +74,13 @@ private AuthorizedMethod(ServiceDescriptor... serviceDescriptor) {
         }
 
         public GrpcServiceAuthorizationConfigurer.Registry hasAnyAuthority(String... authorities) {
-            for (String auth : authorities) {
-                GrpcServiceAuthorizationConfigurer.this.registry.map(auth, methods);
+            if (authorities.length == 0) {
```
Comment (jvmlet) [inline grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/security/GrpcServiceAuthorizationConfigurer.java:77]: Please move this logic to `processSecuredAnnotation()` method (call `authenticated()` if roles array is empty).
The reason for this is semantic

Comment (jvmlet) [reply grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/security/GrpcServiceAuthorizationConfigurer.java:77]: OK, got it.
 I've add comment, please also wright few words about this behavior in README and add test for this case.
Thanks.

Comment (CleverUnderDog) [reply grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/security/GrpcServiceAuthorizationConfigurer.java]: Could you please add the "normal" value behavior to the readme. I don't fully understand it yet.

**Comment by jvmlet** [comment]
[4.2.3](https://github.com/LogNet/grpc-spring-boot-starter/releases/tag/v.4.2.3) is out

**Pull request merged**